### PR TITLE
[ENH] Remove redundant entity definitions on MRI page

### PR DESCRIPTION
**Pull request by tsalo**
Related to #612.

This PR removes any paragraphs describing entities, wherein the description does not differ appreciably from, or expand upon, the official entity definition in the schema, from the MRI page.

### CODE REVIEW

File: tools/schemacode/bidsschematools/render.py [hunk 1]
```diff
@@ -477,7 +477,10 @@ def make_filename_template(
 
 
 def append_filename_template_legend(text, pdf_format=False):
+    """Append a legend to filename templates."""
     legend = """
+- For more information about filename elements (for example, entities, suffixes, extensions),
+  follow the links embedded in the filename template.
```
Comment (effigies) [inline tools/schemacode/bidsschematools/render.py:483]: This will probably be confusing in the PDF build.

Comment (tsalo) [reply tools/schemacode/bidsschematools/render.py:486]: Done!

**Review by Remi-Gau — APPROVED** [review]
LGTM

Thanks for this refactoring @tsalo 

Had a quick look I think that there may be a paragraph or 2 we could similarly remove from PET or microscopy in another PR.

**Comment by tsalo** [comment]
>Had a quick look I think that there may be a paragraph or 2 we could similarly remove from PET or microscopy in another PR.

Absolutely! I definitely want to work through the rest of the spec to remove anything unnecessary.

**Review by sappelhoff — APPROVED** [review]

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**